### DISCUSSION TITLE
Integrate Opire

### DISCUSSION THREAD

**Discussion by nabby27** [FreeTubeApp/FreeTube#4824] [General]
Hi everyone, together with a friend we recently launched [Opire](https://opire.dev). It is a developer rewards platform where anyone can create rewards. This tool aims to help free source projects evolve faster and developers can have economic incentives to improve the software.

Rewards can be accumulated and thus create larger rewards between several people (or one person several times). In addition, maintainers can take a percentage of these rewards, in this way we help continue with the project and all parties win.

We believe that FreeTube would be an ideal project to introduce Opire, the only thing you have to do is install the Opire application in the GitHub repository to start using the commands. You can see more information in the [documentation](https://docs.opire.dev)

We would like to contribute by creating a reward in the issue that you consider best.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @nabby27 we appreciate that you have reached out to us but we are not interested at the moment. 

Good luck with your project and who knows maybe we will cross paths again in the future.
